docs/reusable/pocket-edition-pitch/_index.md

```diff
@@ -0,0 +1,16 @@
+<details>
+  <summary>Would you rather run Snowplow locally?</summary>
+  <p>🎉 We like the way you think! 😉</p>
+  <form name="pocket-edition" method="POST" data-netlify="true">
+    <p>
+      Could you tell us more about what you have in mind?<br/>
+      <textarea name="use-case" rows="4" cols="50" placeholder="I’d like to..."></textarea>
```

Comment:
![image](https://user-images.githubusercontent.com/8820009/195210872-20bbe701-314f-4340-b735-2b0376550059.png)

I think a little css might be required, it overflows the width of the screen on Safari.

Reply:
Done